src/mk.sh

```diff
@@ -639,6 +639,10 @@ function mk_build()
     esac
   done
 
+  if [ -f "$(join_path "$PWD" '/.config')" ]; then
```

Comment:
Why did you add this `/` before `.config`?

Reply:
Oversight, thanks.